crates/turbo-trace/src/tracer.rs

```diff
@@ -211,15 +211,36 @@ impl Tracer {
                     debug!("built in: {:?}", err);
                 }
                 Err(err) => {
-                    // Try to resolve the import as a type import via `@/types/<import>`
-                    let type_package = format!("@types/{}", import);
-                    let resolved_type_import = resolver
-                        .resolve(file_path, type_package.as_str())
+                    if !import.starts_with(".") {
```

Comment:
Is this a check against `./`?

Reply:
Yeah. I don't *think* packages can start with `.`?